pygmt/modules.py

```diff
@@ -88,14 +90,25 @@ def info(fname, **kwargs):
         Report the min/max of the first (0'th) column to the nearest multiple
         of dz and output this as the string *-Tzmin/zmax/dz*.
     """
-    if not isinstance(fname, str):
-        raise GMTInvalidInput("'info' only accepts file names.")
+    kind = data_kind(table)
+    with Session() as lib:
+        if kind == "file":
+            file_context = dummy_context(table)
+        elif kind == "matrix":
+            _table = np.asanyarray(table)
```

Comment:
Just FYI, this line converts `pandas.DataFrame` into a numpy.ndarray, while allowing numpy.ndarray objects to just pass through. A bit easier than using `_table = table.values if hasattr(table, values) else table`, and I think this allows for Python lists to work too (though I haven't tested it).

See also:
- https://stackoverflow.com/questions/14415741/what-is-the-difference-between-numpys-array-and-asarray-functions
- https://stackoverflow.com/questions/13187778/convert-pandas-dataframe-to-numpy-array